client/src/components/Layout/Navbar/NavbarItem.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import s from './NavbarItem.module.less';
+
+type Item = {
+  url: string;
+  label: string;
+  withDot?: boolean;
+};
+
+const NavbarItem: React.FC<Item> = (props) => {
+  return (
+    <>
+      <NavLink
+        to={props.url}
```

Comment:
Add `end` prop, so it behaves properly with nested routes (no two routes active at the same time)